src/components/organisms/MessageBox/MessageBox.tsx

```diff
@@ -27,6 +27,20 @@ const MessageBox: React.FC = () => {
     [alert]
   );
 
+  const modify = (title: string) => {
```

Comment:
Let's create a new file `src/utils/notification.ts` and move this function there so we don't duplicate the code.
I also suggest renaming the function name to `enhanceErrorMessage` and the param to `message`.